### ISSUE TITLE
Inverse Loss Spike issue

### ISSUE THREAD

**Issue by LaBaZh** [xiaoachen98/Open-LLaVA-NeXT#16]
When finetuning pretrained Open-LLaVA-Next on the mixture data, I encoutered inverse loss spike issue. Is such issue caused by the mixture form of data? Is it ok to have such curve during the finetuning?
<img width="426" alt="image" src="https://github.com/user-attachments/assets/2f08ef46-8dd9-4e64-8d61-26cb694cf836">

**Comment by Cooperx521** [comment]
It is very normal and also occurs in the original llava1.5, likely due to the pure text content, although I have not validated it.
